src/exchange/billing/account-renew/renew.html

```diff
@@ -0,0 +1,117 @@
+<div class="px-5 pt-5" data-ui-view>
+    <oui-back-button data-on-click="$ctrl.goBack()">
+        <span data-translate="exchange_update_billing_action_title"></span>
+    </oui-back-button>
+
+    <oui-stepper
+        data-on-finish="$ctrl.submit()">
+        <oui-step-form
+            data-header="{{::'exchange_update_billing_button_title' | translate}}"
+            data-description="{{::'exchange_update_billing_periode_intro' | translate}}"
+            data-loading="!$ctrl.exchange"
+            data-on-cancel="$ctrl.goBack()"
+            data-valid="$ctrl.buffer.hasChanged"
+            data-prevent-next>
+
+            <oui-message class="mb-5"
+                data-type="warning"
+                data-ng-if="!$ctrl.loading && $ctrl.bufferedAccounts && $ctrl.bufferedAccounts.list.messages.length > 0">
+                <strong data-translate="exchange_tab_ACCOUNTS_warning"></strong>
+                <span data-translate="exchange_tab_ACCOUNTS_partial"></span>
+                <ul class="mt-3">
```

Comment:
Keeping the same direction when applying some margins could enforce UI harmony.

Why not simply wrap the two parent element inside a `<p>` tag? (Which has by default a margin).

Reply:
Updated